var/spack/repos/builtin/packages/py-gmsh-interop/package.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyGmshInterop(PythonPackage):
+    """Interoperability between Python and Gmsh"""
+
+    homepage = "https://documen.tician.de/gmsh_interop"
+    git = "https://github.com/inducer/gmsh_interop"
+
+    maintainers = ["cgcgcg"]
+
+    version("2021.1.1", commit="ce5515be82e1f9d13ca638203269c516bc113856")
+
+    depends_on("py-setuptools", type="build")
+    depends_on("py-packaging", type="build")
```

Comment:
Not listed as a dep

Reply:
`packacking` listed in 
https://github.com/inducer/gmsh_interop/blob/568fac4be23f2f1f3ef1c4167a9ae75f4f0d3e70/setup.py#L40
Or was that not what you meant?

Reply:
That's a dependency for main, not a dependency for 2021.1.1: https://github.com/inducer/gmsh_interop/blob/v2021.1.1/setup.py

Reply:
Ah, shoot. My bad. Lemme go ahead and fix this (and probably all other packages).